### ISSUE TITLE
No performance gain after reloading policy. 

### ISSUE THREAD

**Issue by jc-bao** [jc-bao/policy-adaptation-survey#6]
<img width="704" alt="image" src="https://user-images.githubusercontent.com/60093981/209648990-43199014-bee6-464d-8ad0-9930ac31f918.png">

**Comment by jc-bao** [comment]
Using 
```agent.act.load_state_dict(loaded_agent['actor'].state_dict())``` 
to load policy rather than 
```agent.act = loaded_agent['actor']```
where the policy gradient is disabled.
